geoportailv3/static/js/infobar/projectionselector.html

```diff
@@ -0,0 +1,6 @@
+ <select ng-model="ctrl.projection" 
+                ng-options="opt as opt.label for opt in ::ctrl.projectionOptions" ng-change="ctrl.switchProjection()"></select>
+<div class="mouse-coordinates"></div>
```

Comment:
To use `ngeo-control` you will need the following here for the "mouse-coordinates" div:

```html
<div class="mouse-coordinates" ngeo-control="ctrl.mousePositionControl" ngeo-control-map="ctrl.map"></div>
```